.github/workflows/sanitizers.yml

```diff
@@ -0,0 +1,49 @@
+name: sanitizers
+
+on: [push, pull_request]
+
+jobs:
+  address:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Inspect environment
+      run: |
+        whoami
+        gcc --version
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        DEBIAN_FRONTEND='noninteractive' sudo apt-get install -qy meson libtool libglib2.0-dev libcurl3-dev libssl-dev libdbus-1-dev libjson-glib-dev libfdisk-dev libnl-genl-3-dev dbus-x11 clang-tools
+
+    - uses: actions/checkout@v3
+
+    - name: Run meson
+      run: |
+        meson setup build -Db_sanitize=address
+        meson compile -C build
+
+    - name: Run selected tests
+      run: |
+        export G_SLICE=always-malloc G_DEBUG=gc-friendly
```

Comment:
Maybe we should add some hints in the commit message about why these are required or where this is inspired from?